app/scripts/lib/snap-keyring/utils/isBlockedUrl.test.ts

```diff
@@ -6,11 +6,10 @@ describe('isBlockedUrl', () => {
   const messenger = new ControllerMessenger();
   const phishingControllerMessenger = messenger.getRestricted({
     name: 'PhishingController',
+    allowedActions: [],
```

Comment:
Prior to `@metamask/base-controller` 5.0.0, `allowedActions` and `allowedEvents` were optional if no actions or events were allowlisted; now these arguments are required. See changelog: https://github.com/MetaMask/core/blob/main/packages/base-controller/CHANGELOG.md#changed-2